{"uuid": "99821a48-691c-4b55-a9e3-0fb5243e5ae2", "children": ["4b395ee6-8299-4c43-905f-396f65b0d432"], "befores": [{"name": "oneTimeSetUp", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: 'geckodriver' executable needs to be in PATH. \n\n", "trace": "  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\tests\\conftest.py\", line 16, in oneTimeSetUp\n    driver = wdf.getWebDriverInstance()\n  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\base\\webdriverfactory.py\", line 41, in getWebDriverInstance\n    driver = webdriver.Firefox()\n  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\venv\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 164, in __init__\n    self.service.start()\n  File \"D:\\Python Frameworks selenium\\Python-Test-Automation-Framework\\venv\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 81, in start\n    raise WebDriverException(\n"}, "start": 1686467316201, "stop": 1686467316226}], "start": 1686467316201, "stop": 1686467316334}